pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/QuickstartRunner.java

```diff
@@ -75,6 +75,8 @@
   private final boolean _enableTenantIsolation;
   private final String _authToken;
   private final Map<String, Object> _configOverrides;
+  private final String _zkAddress;
```

Comment:
call it `_externalZkAddress`? And add comment that if this is set, Quickstart won't start an embedded zk.

Reply:
Fixed